lib/site_prism/addressable_url_matcher.rb

```diff
@@ -35,7 +36,7 @@ def mappings(url)
     def matches?(url, expected_mappings = {})
       actual_mappings = mappings(url)
       if actual_mappings
-        expected_mappings.empty? ? true : all_expected_mappings_match?(expected_mappings, actual_mappings)
+        expected_mappings.empty? || all_expected_mappings_match?(expected_mappings, actual_mappings)
```

Comment:
If you want to be overly clever you could do something like: `actual_mappings && (expected_mappings.empty? || all_expected_mappings_match?(expected_mappings, actual_mappings)) || false`  (but that's likely not so easy to grok.)

Reply:
Not going to lie, this one I actually didn't debug. I just know this code is semantically similar with one less conditional